pkg/daemon/controller/node.go

```diff
@@ -304,7 +304,6 @@ func (r *nodeReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	if err := nodeController.Watch(&source.Kind{Type: &corev1.Node{}},
 		&fixedKeyHandler{key: ActionReconcileNode},
 		&predicate.ResourceVersionChangedPredicate{},
-		&predicate.LabelChangedPredicate{},
```

Comment:
if all we needed is in annotations, how about replacing LabelChangedPredicate with AnnotationChangePredicate?

Reply:
There already has one, https://github.com/alibaba/hybridnet/pull/225/files#diff-bc1d985df6874b40c6a27274b7d19089f4624ce9d94d481cb60f44d7be9047b2L309.

Reply:
Fixed